app/src/main/kotlin/com/wire/android/ui/userprofile/self/SelfUserProfileScreen.kt

```diff
@@ -440,6 +449,20 @@ private fun CurrentSelfUserStatus(
     }
 }
 
+@Composable
+private fun ManageTeamButton(
+    onManageTeamClick: (String) -> Unit
+) {
+    val teamManagementLink = stringResource(R.string.url_team_management_login)
```

Comment:
this is wrong the team management link should be fetched from the self user server config

Reply:
Done !